wordpress-mobile/WordPress-iOS
# Issue/7878 fix editor bars with long titles

Fixes #7878 

This PR fixes the issues with long names of blogs by setting the `ContentHuggingPriority` on components of the navigation bar on iOS11. This keeps the structure and layout of the navigation bar the same has before and avoid the use of the title view.

<img width="882" alt="screen shot 2017-10-12 at 12 38 47" src="https://user-images.githubusercontent.com/651601/31494246-5d725750-af4a-11e7-9fc5-feda77946f28.png">
<img width="504" alt="screen shot 2017-10-12 at 12 38 43" src="https://user-images.githubusercontent.com/651601/31494247-5d931828-af4a-11e7-9f19-a475bf204888.png">


To test:  
 - Open the editor on a blog with a long name
 - Check that the navigation bar shows correctly and the blog name it's cropped correctly if needed.
 - Test on different devices (iPhone SE, iPhone 8+, iPad) and multitasking modes.
 - Test on iOS10 and iOS11

Thanks to @frosty for the nudge to the right direction.
